workitem/enum_type.go

```diff
@@ -5,12 +5,29 @@ import (
 	"reflect"
 
 	"github.com/fabric8-services/fabric8-wit/convert"
+	errs "github.com/pkg/errors"
 )
 
 type EnumType struct {
-	SimpleType
-	BaseType SimpleType
-	Values   []interface{}
+	SimpleType       `json:"simple_type"`
+	BaseType         SimpleType    `json:"base_type"`
+	Values           []interface{} `json:"values"`
+	RewritableValues bool          `json:"rewritable_values"`
+}
+
+// Ensure EnumType implements the FieldType interface
+var _ FieldType = EnumType{}
+var _ FieldType = (*EnumType)(nil)
+
+// DefaultValue implementes FieldType
+func (t EnumType) DefaultValue(value interface{}) (interface{}, error) {
+	if value != nil {
+		return value, nil
+	}
+	if len(t.Values) <= 0 {
+		return nil, errs.Errorf("enum has no values")
+	}
+	return t.Values[0], nil
```

Comment:
This makes the first Enum value the Default for fields that are not specified?

Reply:
Yes. It is the default behavior for all enum types and currently only the enum values have an implicit default type.